database/Seed.js

```diff
@@ -21,11 +21,26 @@ function createImageObjects() {
   return result;
 }
 
-for (let i = 0; i < 100; i += 1) {
-  const obj = {
-    id: i,
-    name: faker.commerce.productName(),
-    imgObjects: createImageObjects(),
-  };
-  itemModel.item.create(obj);
+async function addData() {
+  const promises = [];
+  for (let i = 0; i < 100; i += 1) {
+    const obj = {
+      id: i,
+      imgObjects: createImageObjects(),
+    };
+    // eslint-disable-next-line no-await-in-loop
+    const newItem = await itemModel.item.create(obj);
+    promises.push(newItem);
+  }
+  return promises;
 }
+
+addData()
+  .then(() => {
+    console.log('Database complete!');
+    process.exit();
+  })
```

Comment:
Good modification to allow user to see the process is complete by exiting the script run.